waku/waku.go

```diff
@@ -110,7 +110,7 @@ type Waku struct {
 
 // init initialises the waku package
 func init() {
-	initRLPKeyFields()
+	v0.Init()
 }
```

Comment:
This isn't needed now, see https://github.com/status-im/status-go/pull/1947/files#r415753463 . If we are moving the variable initialisation into the version packages, then they can be loaded automatically when the specific package is accessed.